CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Samuel Giddins](https://github.com/segiddins)
   [#3101](https://github.com/CocoaPods/CocoaPods/issues/3101)
 
+* Add environment variable `COCOAPODS_SKIP_UPDATE_MESSAGE` to disable new
```

Comment:
this is an `Enhancement`, which is the section previous to `Bug Fixes` :)
Also, please make sure to end the commit message with a period and two trailing spaces.